kumascript/macros/ifmethod.ejs

```diff
@@ -6,13 +6,7 @@ var lang = env.locale;
 var destNewType = lang + '/docs/XPCOM_Interface_Reference/' + $0;
 var destOldType = lang + '/docs/' + $0;
 var apiString = $0 + '.' + $1 + '()';
-if(wiki.pageExists(destNewType)) { /* the page exists from the dekiwiki age */
-    %><code><%-(web.link(wiki.uri(destNewType) + '#' + $1 + '()', apiString))%></code><%
-} else if (wiki.pageExists(destOldType)) { /* the page exists from the mediawiki age */
-    %><code><%-(web.link(wiki.uri(destOldType) + '#' + $1 + '()', apiString))%></code><%
-} else { /* the page doesn't exist */
-  var targeturi = "https://developer.mozilla.org/Article_not_found?uri=" + destNewType +
-        "&ident=" + $0;
-  %><code><a rel="internal" href="<%=targeturi%>" class="new"><%=apiString%></a></code><%
-}
-%>
+/* the page doesn't exist */
```

Comment:
What page?
I think you copied `else { ... }` block and then the code comment used to make sense.

Reply:
Yep, I think it used to make sense but now it's clear the comment doesn't need to be there.